### DISCUSSION TITLE
v3 - 'createReceiptShipment' issue

### DISCUSSION THREAD

**Discussion by FatihGokmen** [etsy/open-api#1122] [Q&A]
I send my greetings to everyone

We send shipping shipping codes to Etsy via the API, but strangely they are not reflected in the order's shipping number and shipping company fields. It also keeps adding the post field to every API message we send.

**Comment by alex-monte** [comment]
We have a problem that appears to be random. We send the name of the carrier and the tracking number via API but it fails sometimes. 'carrier_name' is always reported correctly, but 'tracking_code' is sometimes not reflected as reported.

**Reply by Gareth064** [reply]
What is the full error you are getting back?

**Reply by alex-monte** [reply]
I don't get any error in the response

**Reply by Gareth064** [reply]
If it's just a 500 response then no one here will be able to help as that's an internal error happening on Etsy end.

You can email them to ask them to investigate.
